src/Components/ScrivitoExtensions/ColumnsEditorTab.js

```diff
@@ -238,6 +238,10 @@ const Alignment = Scrivito.connect(({ widget }) => {
       </div>
     </React.Fragment>
   );
+
+  function setAlignment(newAlignment) {
```

Comment:
* [ ] FYI naming it `alignment` instead of `newAlignment` would simplify the code a bit